cabextract.yaml

```diff
@@ -0,0 +1,39 @@
+package:
+  name: cabextract
+  version: 1.11
+  epoch: 0
+  description: "Tool for extracting Microsoft cabinet files"
+  copyright:
+    - license: GPL-2.0-or-later
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - libmspack-dev
+
+pipeline:
+  - uses: fetch
+    with:
+      uri: https://www.cabextract.org.uk/cabextract-${{package.version}}.tar.gz

```

Comment:
I think we can switch this to `git-checkout` https://github.com/kyz/libmspack

Reply:
hmm that didn't work as I suspect the upstream release is adding extra configure files.  Let's ignore this comment.